src/Files.App/Converters/Converters.cs

```diff
@@ -85,37 +85,6 @@ public abstract class ValueConverter<TSource, TTarget>
 		}
 	}
 
-	/// <summary>
-	/// The base class for converting instances of type T to object and vice versa.
-	/// </summary>
-	public abstract class ToObjectConverter<T>
```

Comment:
We should probably keep these, what do you think?

Reply:
Let's restore them in that case. I tried to not remove unused code that would have real value, but this one has eluded me.

Reply:
Added back.